Zend/zend_API.c

```diff
@@ -3723,58 +3724,151 @@ ZEND_API int zend_declare_property_stringl(zend_class_entry *ce, const char *nam
 }
 /* }}} */
 
-ZEND_API int zend_declare_class_constant(zend_class_entry *ce, const char *name, size_t name_length, zval *value) /* {{{ */
+ZEND_API int zend_declare_class_constant_ex(zend_class_entry *ce, zend_string *name, zval *value, int access_type, zend_string *doc_comment) /* {{{ */
```

Comment:
@bwoebi @Danack 

This is fixed in the RFC! I will be pushing more commits to fix the (many) issues people have found so far. Adding the ZEND_ACC_PUBLIC check for interfaces.